playground/mocks/data/idp/idx/authenticator-enroll-ov-via-email.json

```diff
@@ -156,7 +156,8 @@
                       },
                       {
                         "name": "channel",
-                        "required": false,
+                        "required": true,
+                        "visible": true,
```

Comment:
Why is this visible: true now?

Reply:
Channel selection needs to be visible. I have it as visible everywhere else and just dint update this json earlier.